ReClass.NET/Forms/PluginForm.cs

```diff
@@ -43,9 +43,13 @@ internal PluginForm(PluginManager pluginManager)
 
 			UpdatePluginDescription();
 
-			// Native Methods Tab
+            // Native Methods Tab
 
-			functionsProvidersComboBox.Items.AddRange(Program.CoreFunctions.FunctionProviders.ToArray());
+            string[] arrayOfFunctions = Program.CoreFunctions.FunctionProviders.ToArray();
+            var funcIndexStr = Program.CoreFunctions.functionsRegistry.FirstOrDefault(p => p.Value == Program.CoreFunctions.CurrentFunctions);
+
+			functionsProvidersComboBox.Items.AddRange(arrayOfFunctions);
+            functionsProvidersComboBox.SelectedIndex = funcIndexStr.Value == null ? 0 : Array.IndexOf(arrayOfFunctions, funcIndexStr.Key);
```

Comment:
Here is a possible NPE. FirstOrDefault may return null, but you check `funcIndexStr.Value == null`.

Reply:
i thought that FirstOrDefault does not return null, and that it will at least be an empty object?